sdk/js/src/token_bridge/attest.ts

```diff
@@ -240,37 +245,23 @@ export async function attestFromAlgorand(
   return txs;
 }
 
-/**
- * Attest an already created asset
- * If you create a new asset on near and want to transfer it elsewhere,
- * you create an attestation for it on near... pass that vaa to the target chain..
- * submit it.. then you can transfer from near to that target chain
- * @param client An Near account client
- * @param coreBridge The account for the core bridge
- * @param tokenBridge The account for the token bridge
- * @param asset The account for the asset
- * @returns [sequenceNumber, emitter]
- */
```

Comment:
Was it better to remove all the documentation instead of updating it?

Reply:
In my opinion the parameter names are self-explanatory, but am open to adding the comments if necessary